python/lsst/obs/base/_fitsRawFormatterBase.py

```diff
@@ -166,7 +166,10 @@ def readMetadata(self):
 
     def stripMetadata(self):
         """Remove metadata entries that are parsed into components."""
-        self._createSkyWcsFromMetadata()
+        try:
+            lsst.afw.geom.stripWcsMetadata(self.metadata)
+        except TypeError as e:
+            log.warning("Cannot create a valid WCS from metadata: %s", e.args[0])
```

Comment:
This warning should no longer exist because this is stripping (which should always work) and not creating a WCS.